gym_electric_motor/physical_systems/voltage_supplies.py

```diff
@@ -62,3 +66,49 @@ def __init__(self, u_nominal=600.0, **__):
     def get_voltage(self, *_, **__):
         # Docstring of superclass
         return self._u_nominal
+
+
+class RCVoltageSupply(VoltageSupply):
+    """Voltage supply moduled as RC element"""
+    
+    def __init__(self, u_nominal=600.0, supply_parameter={'R':1,'C':4e-3}, **__):

```

Comment:
Mutable variables (the dictionary) should not be used as default parameters in function headers. (See: http://google.github.io/styleguide/pyguide.html Section: 2.12.4 Decision)

Instead use: 
`supply_parameter=None` as default value
and 
`supply_parameter=supply_parameter or {'R': 1, 'C': 4e-3}`
inside the function